api/src/test/java/org/openmrs/module/datafilter/UtilTest.java

```diff
@@ -52,11 +52,13 @@
 import org.openmrs.module.datafilter.registration.HibernateFilterRegistration;
 import org.openmrs.util.OpenmrsClassLoader;
 import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 import org.w3c.dom.Document;
 
 @RunWith(PowerMockRunner.class)
+@PowerMockIgnore({ "org.xml.*" })
```

Comment:
Avoids linkage error with XMLReader

Reply:
I'd think this happens when loggers are getting instantiated in the mocked classes, disabling static initialization for specific classed would probably be better

Reply:
I've tried @SuppressStaticInitializationFor but with no success so far

Reply:
ok

Reply:
FYI SuppressStaticInitializationFor takes fully qualified classnames

Reply:
Yes, not sure if it's the right thing to do but, tried that with `org.openmrs.module.datafilter.Util`. The exception occurs on this Util class, here:
`Transformer transformer = transformerFactory.newTransformer(new StreamSource(xslt));`
But, then got some null pointer exceptions as there's a lot of things statically initialized on that class.